t/admin/secrets.t

```diff
@@ -175,11 +165,6 @@ passed
         content_by_lua_block {
             local t = require("lib.test_admin").test
             local etcd = require("apisix.core.etcd")
-            local res = assert(etcd.get('/secrets/vault/test1'))
-            local prev_create_time = res.body.node.value.create_time
-            assert(prev_create_time ~= nil, "create_time is nil")
-            local prev_update_time = res.body.node.value.update_time
-            assert(prev_update_time ~= nil, "update_time is nil")
             ngx.sleep(1)
```

Comment:
can you consdier add new test for compatibility ?

Reply:
I don't think it's necessary to add it because there was no create_time and update_time to guide the user to create a resource in the previous admin api documentation. Forward compatibility is not guaranteed. @Sn0rt